### PR TITLE
Refactored to pass down resource obj to operation.js

### PR DESCRIPTION
**Pull request by jackyzhen**

### CODE REVIEW

File: src/application/components/Application.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ export class Application extends Component {
       } = this.props.params;
 
       const operation = utils.getOperation(resource, method, path, service);
+      const resourceObj = utils.getResource(resource, service);
```
Comment (kalmanb) [inline src/application/components/Application.js]: Call this resource. eg
const resource = utils.getResource(this.props.params.resource, service)
Remove resource from the destructure

File: src/utils.js [hunk 2]
```diff
@@ -106,10 +106,18 @@ const getOperation = (type: string, method: Method, path: string, service: Servi
   if (!operation) {
     throw new Error(`Operation not found: ${type}`);
   }
-  operation.resourceDescription = resource.description;
   return operation;
 };
 
+const getResource = (type: string, service: Service): Resource => {
```
Comment (kalmanb) [inline src/utils.js]: Refactor getOperation to use this - DRY

File: src/application/components/Application.js (hunk 1 above)
Comment (jackyzhen) [inline src/application/components/Application.js]: how about

```const {
        method,
        path,
        resource: resourceType,
        applicationKey,
        organizationKey,
      } = this.props.params;

      const operation = utils.getOperation(resourceType, method, path, service);
      const resource = utils.getResource(resourceType, service);

      return (
        <LoadingOverlay isLoaded={this.props.loaded}>
          <Operation
            service={service}
            importedServices={importedServices}
            operation={operation}
            applicationKey={applicationKey}
            organizationKey={organizationKey}
            resource={resource}
            method={method}
            path={path}
          />
        </LoadingOverlay>```

File: src/utils.js (hunk 2 above)
Comment (jackyzhen) [inline src/utils.js]: 👍

**Comment by jackyzhen** [comment]
@kalmanb what do you think of this refactor?

**Comment by kalmanb** [comment]
LGTM

**Pull request merged**